documentation/milestones/individual_milestones/developer_tools.md

```diff
@@ -0,0 +1,221 @@
+  - [ ] Code which fails any of these checks will be blocked from merging into `main`
+- [ ] **Security checks** are live and meet the following conditions:
+  - [ ] At least 1 (test) security vulnerability report has been submitted
+  - [ ] Maintainers are notified within 1 hour of a vulnerability being reported within the grants API codebase
+  - [ ] Maintainers are notified within 72 hours of a vulnerability being reported on an upstream dependency
+  - [ ] Security checks are running on every push to the remote GitHub repository
+  - [ ] Code which fails any of these security checks is blocked from merging into `main`
+- [ ] **Dependency management** is live and meets the following conditions:
+  - [ ] At least 2 upstream dependencies have been added to the project
+  - [ ] Compatible versions of upstream dependencies are automatically detected when a new dependency is added
+  - [ ] Maintainers are notified when a new minor or major version of an upstream dependency is available
+- [ ] **Config & secrets management** is live and meets the following conditions:
+  - [ ] At least 1 configuration variable has been added to the project
+  - [ ] This config variable has different values when it is running in different environments (e.g. dev, staging, and prod)
+  - [ ] At least 2 internal developers can access a shared value of this variable locally
+  - [ ] Developers can change the value of this config variable in their local environment without changing it for other developers
+  - [ ] An integration test has been created to validate that this variable is injected at runtime
+
+### Proposed metrics for measuring goals/value/definition of done
+
+- Test framework metrics
+  - Percentage of code covered by unit tests
+  - Percentage of code covered by integration tests
+  - Total runtime of all unit tests
+  - Total runtime of all integration tests
+- Code quality checks metric: Total runtime of all code quality checks
+- Security checks metrics
+  - Total runtime of all security checks
+  - Number of security vulnerabilities not caught until production
+  - Number of days it takes to resolve a security vulnerability reported in the grants API project
+  - Number of days it takes to update an upstream dependency once a version which resolves a CVE has been released
+- Dependency management metrics
+  - Number of days it takes to update an upstream dependency once a new minor or major version has been released
+  - Number of dependencies that received funding through this project
+- Config & secrets management metric: Average age across all tokens/secrets
+
+### Destination for live updating metrics
+
+Not yet known
+
+## Planning
+
+### Assumptions & dependencies
+
+What capabilities / milestones do we expect to be in place at the beginning of work
+on this milestone?
+
+- [ ] **API Plan & frontend Plan:** Choosing a set of developer tools for will be depend heavily on the language selected for both the backend and the frontend.
```

Comment:
i think 'for will be' is a typo?

Reply:
Yes! Good catch, I fixed this in the last commit